src/Shared/LayoutRenderers/AspNetRequestServerVariableLayoutRenderer.cs

```diff
@@ -0,0 +1,73 @@
+#if !ASP_NET_CORE || ASP_NET_CORE3
+using System.Text;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using NLog.Config;
+#if !ASP_NET_CORE
+using System.Web;
+#else
+using Microsoft.AspNetCore.Http.Features;
+#endif
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Server Variable.
+    /// </summary>
+    /// <remarks>
+    /// Use this layout renderer to insert the value of the specified Server Variable
+    /// </remarks>
+    /// <example>
+    /// <para>Example usage of ${aspnet-request-servervariable}:</para>
+    /// <code lang="NLog Layout Renderer">
+    /// ${aspnet-request-servervariable:ServerVariable=v}
+    /// </code>
+    /// </example>
+    [LayoutRenderer("aspnet-request-servervariable")]
+    public class AspNetRequestServerVariableLayoutRenderer : AspNetLayoutRendererBase
+    {
+
+        /// <summary>
+        /// Gets or sets the ServerVariables item to be rendered.
+        /// </summary>
+        [RequiredParameter]
+        [DefaultParameter]
+        public string ServerVariable { get; set; }
+
+/// <summary>
+/// Renders the specified ASP.NET Request variable and appends it to the specified <see cref="StringBuilder" />.
+/// </summary>
+/// <param name="builder">The <see cref="StringBuilder" /> to append the rendered data to.</param>
+/// <param name="logEvent">Logging event.</param>
+protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
```

Comment:
Please reformat code to fix proper indent.

Reply:
Complete